src/Constants.js

```diff
@@ -497,37 +666,553 @@ export const SPIRIT_TEMPLE = [
+  "Kakariko-Bazaar-Item-8",
+];
+
+export const ZORA_SHOP = [
+  "Zora-Shop-Item-1",
+  "Zora-Shop-Item-2",
+  "Zora-Shop-Item-3",
+  "Zora-Shop-Item-4",
+  "Zora-Shop-Item-5",
+  "Zora-Shop-Item-6",
+  "Zora-Shop-Item-7",
+  "Zora-Shop-Item-8",
+];
+
+export const GORON_SHOP = [
+  "Goron-Shop-Item-1",
+  "Goron-Shop-Item-2",
+  "Goron-Shop-Item-3",
+  "Goron-Shop-Item-4",
+  "Goron-Shop-Item-5",
+  "Goron-Shop-Item-6",
+  "Goron-Shop-Item-7",
+  "Goron-Shop-Item-8",
+];
+
+export const DODONGOS_CAVERN_SCRUB = [
+  "DC-Deku-Scrub-Deku-Nuts",
+  "DC-Deku-Scrub-Deku-Sticks",
+  "DC-Deku-Scrub-Deku-Seeds",
+  "DC-Deku-Scrub-Deku-Shield",
+];
+
+export const JABU_JABUS_BELLY_SCRUB = [
+"Jabu-Deku-Scrub-Deku-Nuts",
+];
+
+export const GANONS_CASTLE_SCRUB = [
+  "GC-Deku-Scrub-Bombs",
+  "GC-Deku-Scrub-Arrows",
+  "GC-Deku-Scrub-Red-Potion",
+  "GC-Deku-Scrub-Green-Potion",
+];
+
+export const GS_DEKU_TREE_SCRUB_MQ = [
+"DT-MQ-Deku-Scrub-Deku-Shield",
+];
+
+export const DODONGOS_CAVERN_SCRUB_MQ = [
+  "DC-MQ-Deku-Scrub-Deku-Sticks",
+  "DC-MQ-Deku-Scrub-Deku-Seeds",
+  "DC-MQ-Deku-Scrub-Deku-Shield",
+  "DC-MQ-Deku-Scrub-Red-Potion",
+];
+
+export const GANONS_CASTLE_SCRUB_MQ = [
+  "GC-MQ-Deku-Scrub-Deku-Nuts",
+  "GC-MQ-Deku-Scrub-Bombs",
+  "GC-MQ-Deku-Scrub-Arrows",
+  "GC-MQ-Deku-Scrub-Red-Potion",
+  "GC-MQ-Deku-Scrub-Green-Potion",
+];
+
+export const ZORAS_RIVER_SCRUB = [
+  "ZR-Grotto-Deku-Scrub-Red-Potion",
+  "ZR-Grotto-Deku-Scrub-Green-Potion",
+];
+
+export const SACRED_FOREST_MEADOW_SCRUB = [
+  "SFM-Grotto-Deku-Scrub-Red-Potion",
+  "SFM-Grotto-Deku-Scrub-Green-Potion",
+];
+
+export const LAKE_HYLIA_SCRUB = [
+  "LH-Grotto-Deku-Scrub-Deku-Nuts",
+  "LH-Grotto-Deku-Scrub-Bombs",
+  "LH-Grotto-Deku-Scrub-Arrows",
+];
+
+export const GERUDO_VALLEY_SCRUB = [
+  "Valley-Grotto-Deku-Scrub-Red-Potion",
+  "Valley-Grotto-Deku-Scrub-Green-Potion",
+];
+
+export const LOST_WOODS_SCRUB = [
+  "LW-Deku-Scrub-Deku-Nuts",
+  "LW-Deku-Scrub-Deku-Sticks",
+  "LW-Grotto-Deku-Scrub-Arrows",
+];
+
+export const DESERT_COLOSSUS_SCRUB = [
+  "Desert-Grotto-Deku-Scrub-Red-Potion",
+  "Desert-Grotto-Deku-Scrub-Green-Potion",
+];
+
+export const DEATH_MOUNTAIN_CRATER_SCRUB = [
+  "DMC-Deku-Scrub-Bombs",
+  "DMC-Grotto-Deku-Scrub-Deku-Nuts",
+  "DMC-Grotto-Deku-Scrub-Bombs",
+  "DMC-Grotto-Deku-Scrub-Arrows",
+];
+
+export const GORON_CITY_SCRUB = [
+  "Goron-Grotto-Deku-Scrub-Deku-Nuts",
+  "Goron-Grotto-Deku-Scrub-Bombs",
+  "Goron-Grotto-Deku-Scrub-Arrows",
+];
+
+export const LON_LON_RANCH_SCRUB = [
+  "LLR-Grotto-Deku-Scrub-Deku-Nuts",
+  "LLR-Grotto-Deku-Scrub-Bombs",
+  "LLR-Grotto-Deku-Scrub-Arrows",
+];
+
+export const POTS = [
+  "Goron-City-Stick-Pot",
+  "Bottom-of-the-Well-Stick-Pot",
+  "Zoras-Domain-Stick-Pot",
+  "Zoras-Domain-Nut-Pot",
+];
+
+export const ENEMIES = [
+  "Deku-Baba-Sticks",
+  "Deku-Baba-Nuts",
+];
+
+export const OTHER = [
+  "Epona",
+  "Spirit-Temple-Nut-Crate",
+  "Gerudo-Fortress-Carpenter-Rescue",
+  "Ganon",
```

Comment:
There's no point in adding this as a location, since the Ganon kill is the end of the game